2024-01-27 15:49:26,062 INFO    MainThread:40197 [wandb_setup.py:_flush():69] setting env: {}
2024-01-27 15:49:26,063 WARNING MainThread:40197 [wandb_setup.py:_flush():69] could not save program above cwd: /Users/ruilu/anaconda3/envs/mlflow-8d9e81b9c264be0055d76a0ebe3d4d0b2aaf0212/bin/pytest
2024-01-27 15:49:26,063 INFO    MainThread:40197 [wandb_setup.py:_flush():69] setting login settings: {}
2024-01-27 15:49:26,063 INFO    MainThread:40197 [wandb_init.py:_log_setup():334] Logging user logs to /Users/ruilu/udacity_repo/exercise_8/wandb/run-20240127_154926-3j2ccqe8/logs/debug.log
2024-01-27 15:49:26,063 INFO    MainThread:40197 [wandb_init.py:_log_setup():335] Logging internal logs to /Users/ruilu/udacity_repo/exercise_8/wandb/run-20240127_154926-3j2ccqe8/logs/debug-internal.log
2024-01-27 15:49:26,064 INFO    MainThread:40197 [wandb_init.py:init():367] calling init triggers
2024-01-27 15:49:26,064 INFO    MainThread:40197 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2024-01-27 15:49:26,065 INFO    MainThread:40197 [wandb_init.py:init():416] starting backend
2024-01-27 15:49:26,065 INFO    MainThread:40197 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-01-27 15:49:26,094 INFO    MainThread:40197 [backend.py:ensure_launched():123] starting backend process...
2024-01-27 15:49:26,104 INFO    MainThread:40197 [backend.py:ensure_launched():127] started backend process with pid: 40508
2024-01-27 15:49:26,106 INFO    MainThread:40197 [wandb_init.py:init():421] backend started and connected
2024-01-27 15:49:26,108 INFO    MainThread:40197 [wandb_init.py:init():461] updated telemetry
2024-01-27 15:49:26,108 INFO    MainThread:40197 [wandb_init.py:init():480] communicating current version
2024-01-27 15:49:26,590 INFO    MainThread:40197 [wandb_init.py:init():485] got version response upgrade_message: "wandb version 0.16.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-01-27 15:49:26,591 INFO    MainThread:40197 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2024-01-27 15:49:26,883 INFO    MainThread:40197 [wandb_init.py:init():518] starting run threads in backend
2024-01-27 15:49:26,935 INFO    MainThread:40197 [wandb_run.py:_console_start():1476] atexit reg
2024-01-27 15:49:26,935 INFO    MainThread:40197 [wandb_run.py:_redirect():1339] redirect: SettingsConsole.REDIRECT
2024-01-27 15:49:26,937 INFO    MainThread:40197 [wandb_run.py:_redirect():1344] Redirecting console.
2024-01-27 15:49:26,937 INFO    MainThread:40197 [redirect.py:install():213] install start
2024-01-27 15:49:26,937 INFO    MainThread:40197 [redirect.py:install():228] install stop
2024-01-27 15:49:26,937 INFO    MainThread:40197 [redirect.py:install():213] install start
2024-01-27 15:49:26,937 INFO    MainThread:40197 [redirect.py:install():228] install stop
2024-01-27 15:49:26,937 INFO    MainThread:40197 [wandb_run.py:_redirect():1390] Redirects installed.
2024-01-27 15:49:26,937 INFO    MainThread:40197 [wandb_init.py:init():542] run started, returning control to user process
2024-01-27 15:49:28,714 INFO    MainThread:40197 [wandb_run.py:_atexit_cleanup():1446] got exitcode: 0
2024-01-27 15:49:28,714 INFO    MainThread:40197 [wandb_run.py:_restore():1418] restore
2024-01-27 15:49:28,714 INFO    MainThread:40197 [redirect.py:uninstall():232] uninstall start
2024-01-27 15:49:28,714 INFO    MainThread:40197 [redirect.py:_stop():287] _stop: stdout
2024-01-27 15:49:28,714 INFO    MainThread:40197 [redirect.py:_stop():293] _stop closed: stdout
2024-01-27 15:49:28,714 INFO    stdout    :40197 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2024-01-27 15:49:28,714 INFO    stdout    :40197 [redirect.py:_pipe_relay():145] relay done done: stdout
2024-01-27 15:49:28,714 INFO    MainThread:40197 [redirect.py:_stop():299] _stop joined: stdout
2024-01-27 15:49:28,714 INFO    MainThread:40197 [redirect.py:_stop():301] _stop rd closed: stdout
2024-01-27 15:49:28,714 INFO    MainThread:40197 [redirect.py:uninstall():236] uninstall done
2024-01-27 15:49:28,715 INFO    MainThread:40197 [redirect.py:uninstall():232] uninstall start
2024-01-27 15:49:28,715 INFO    MainThread:40197 [redirect.py:_stop():287] _stop: stderr
2024-01-27 15:49:28,715 INFO    MainThread:40197 [redirect.py:_stop():293] _stop closed: stderr
2024-01-27 15:49:28,715 INFO    stderr    :40197 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2024-01-27 15:49:28,715 INFO    stderr    :40197 [redirect.py:_pipe_relay():145] relay done done: stderr
2024-01-27 15:49:28,715 INFO    MainThread:40197 [redirect.py:_stop():299] _stop joined: stderr
2024-01-27 15:49:28,715 INFO    MainThread:40197 [redirect.py:_stop():301] _stop rd closed: stderr
2024-01-27 15:49:28,715 INFO    MainThread:40197 [redirect.py:uninstall():236] uninstall done
2024-01-27 15:49:28,852 INFO    MainThread:40197 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 738
  total_bytes: 738
}

2024-01-27 15:49:30,856 INFO    MainThread:40197 [wandb_run.py:_wait_for_finish():1568] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1782
  total_bytes: 1782
}

2024-01-27 15:49:31,936 INFO    MainThread:40197 [wandb_run.py:_show_files():1790] logging synced files
